client/css/editmode.css

```diff
@@ -1,30 +1,67 @@
+
 #addOverlay {
   width: var(--roomWidth);
   height: var(--roomHeight);
   transform: scale(var(--scale));
   transform-origin: left top;
-  flex-direction: row;
   padding: 0;
 }
 
-#addOverlay > .category {
+#gridContainerAddOverlay {
+  display: grid !important;
+  grid-template-areas:         "cards pieces decorative"
+                               "cards pieces decorative"
+                               "cards interactive custom";
+  grid-template-columns: 1fr 3fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  height:100%;
+  margin: 20px;
+  gap: 20px 20px;
+}
+
+#gridContainerAddOverlay > .category {
   background: var(--roomColor);
   width: 100%;
+  height: 100%;
   border-radius: 32px;
-  margin: 20px;
   text-align: center;
 }
 
-#addOverlay > .category > h2 {
+#gridContainerAddOverlay > .category > h2 {
   color: black;
 }
 
-#addOverlay > .category > button {
+#gridContainerAddOverlay > .category > button {
   display: flex;
   margin-left: auto;
   margin-right: auto;
   margin-bottom: 8px;
 }
+
+#gridContainerAddOverlay > .category > div > button {
+  margin-bottom: 8px;
```

Comment:
Seems like every .ui-button on this page wants margin-bottom: 8px, maybe just add single selector to do this, e.g.:
```css
#gridContainerAddOverlay .ui-button {
  margin-bottom: 8px;
}
```

Though this makes me wonder why we don't have `margin: 8px;` (i.e. on all sides) of all `.ui-button`.

Reply:
Done.